### PR TITLE
support KUBECONFIG with multiple kube config files

### PR DESCRIPTION
**Pull request by grounded042**
This change is [Reviewable](https://reviewable.io/reviews/fission/fission/1126)

### CODE REVIEW

**Comment by grounded042** [comment]
this fixes https://github.com/fission/fission/issues/821

File: fission/util/util.go [hunk 1]
```diff
@@ -128,10 +113,31 @@ func KubifyName(old string) string {
...
-	config, err := clientcmd.BuildConfigFromFlags("", kubeConfig)
+// GetKubernetesClient builds a new kubernetes client. If the KUBECONFIG
+// environment variable is empty or doesn't exist, ~/.kube/config is used for
+// the kube config path
+func GetKubernetesClient() (*restclient.Config, *kubernetes.Clientset) {
+	loadingRules := clientcmd.NewDefaultClientConfigLoadingRules()
+
+	kubeConfigPath := os.Getenv("KUBECONFIG")
+	if len(kubeConfigPath) == 0 {
+		home := os.Getenv("HOME")
```
Comment (life1347) [inline fission/util/util.go]: To support windows better, I suggest here we can use `os/user` to get the correct home directory path.
reference: https://stackoverflow.com/a/13004756

Comment (grounded042) [reply fission/util/util.go]: Thanks! I'll take a look.

Comment (grounded042) [reply fission/util/util.go]: @life1347 I changed things up as per the link. Seems like a better cross-platform implementation.

**Comment by life1347** [comment]
LGTM

**Pull request merged**